src/renderer/src/components/song/song-detail/SongDetail.tsx

```diff
@@ -52,8 +67,14 @@ const ProgressBar = () => {
       onValueCommit={handleSeekEnd}
       animate
     >
-      <Slider.Track class="flex h-7 items-center rounded-xl bg-thick-material p-1">
-        <Slider.Range class="block h-5 rounded-l-lg bg-surface" />
+      <Slider.Track
+        class="flex h-7 items-center rounded-xl p-1"
+        style={{ "background-color": props.averageColor }}
```

Comment:
Can you remove the background-color from the track? I think it should only be on the range.

Reply:
If we are going to store a darker border color it might be a nice idea to display the darker color on the slider track, while having the normal color on range?